centreon/src/Core/AdditionalConnectorConfiguration/Domain/Model/VmWareV6/VSphereServer.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+/*
+ * Copyright 2005 - 2023 Centreon (https://www.centreon.com/)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * For more information : contact@centreon.com
+ *
+ */
+
+declare(strict_types=1);
+
+namespace Core\AdditionalConnectorConfiguration\Domain\Model\VmWareV6;
+
+class VSphereServer
+{
+    public function __construct(
+        private readonly string $name,
+        private readonly string $url,
+        private readonly string $username,
+        private readonly string $password
```

Comment:
No property length control?

Reply:
There is no really needs, as when we create an ACC everything is store as a json 
```json
{
    "port": 442,
    "vcenters": [
        {
            "name": "my-vcenter A",
            "url": "https://127.0.0.1/sdk",
            "username": "<encrypted_username>",
            "password": "<encrypted_password>"
        },
        {
            "name": "my-vcenter B",
            "url": "https://127.0.0.2/sdk",
            "username": "<encrypted_username>",
            "password": "<encrypted_password>"
        },
        {
            "name": "my-vcenter C",
            "url": "https://127.0.0.2/sdk",
            "username": "<encrypted_username>",
            "password": "<encrypted_password>"
        }
    ]
}
```